app/views/tags/edit.js.erb

```diff
@@ -0,0 +1,2 @@
+$('[data-behavior~=modal-tag-form]').modal('show');
```

Comment:
should we guard this so this code doesn't run in every single view?

Reply:
We have added a guard in [here](https://github.com/dradis/dradis-ce/pull/1069/files#diff-0ce3ccc401d82b8c2668f62ba39540adf2f26f57b33038c8f9ba27b0117501fe) which make sure that modal-tag-form is only rendered in tags and issues controller views. And edit.js.erb file is only rendered when we have a request in tags_controller#edit.